docker/mysql/Dockerfile

```diff
@@ -0,0 +1,64 @@
+FROM debian:jessie
+
+# add our user and group first to make sure their IDs get assigned consistently, regardless of whatever dependencies get added
+RUN groupadd -r mysql && useradd -r -g mysql mysql
+
+# add gosu for easy step-down from root
+ENV GOSU_VERSION 1.7
+RUN set -x \
```

Comment:
I took this script from docker images hub (https://hub.docker.com/_/mysql/) and didn't change anything, so I'm not sure if it's required or not.